pl_bolts/models/diffeq/neuralode.py

```diff
@@ -0,0 +1,115 @@
+from argparse import ArgumentParser
+
+import pytorch_lightning as pl
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+from torchdyn.models import NeuralDE, DataControl
+
+
+class DepthInvariantNeuralODE(pl.LightningModule):
+    """
+    Vanilla Neural ODE with constant-in-depth parameters.
+    Can be equipped with data control as described in https://arxiv.org/abs/2002.08071
+
+    Example::
+        TO DO
+
+    Example CLI::
+        TO DO
+    """
+
+    def __init__(
+            self,
+            input_dim: int,
+            hidden_dim: int,
+            absolute_tol: float = 1e-4,
+            relative_tol: float = 1e-4,
+            integration_span: Tensor = torch.linspace(0, 1),
+            data_control: str = 'True',
+            lr: float = 0.01,
+            **kwargs
+    ):
+        """
+        Args:
+            input_dim: size of input data
+            hidden_dim: size of hidden state
+            absolute_tol: absolute tolerance of the numerical solver
+            relative_tol: relative tolerance of the numerical solver
+            integration_span: mesh grid of evaluation points for the ODE
+            data_control: data control mechanism as described in https://arxiv.org/abs/2002.08071
+            learning_rate: learning rate for Adam
+
+        """
+        super().__init__()
```

Comment:
Let's raise an exception here with a helpful message for users.
```suggestion
        """
        if not _TORCHDYN_AVAILABLE:  # pragma: no cover
            raise ModuleNotFoundError("You want to use `torchdyn` which is not installed yet.")

        super().__init__()
```